src/index.html

```diff
@@ -12,6 +12,100 @@
   </head>
   <body>
     <h1>HTML Form</h1>
+
+    <form action="https://mate-academy-form-lesson.herokuapp.com/create-application" method="post" onsubmit="onSubmit()">
+      <div class="personal_information">
+        <div class="text_border">Personal information</div>
+        <label>
+          Surname:
+          <input type="text" name="Surname">
+        </label>
+        <br><br>
+        <label>
+          Name:
+          <input type="text" name="Name">
+        </label>
+        <br><br>
+        <label>
+          How old are You?
+          <input id="number" type="number" name="How old are You?" min="12" max="100" value="12">
+        </label>
+        <br><br>
+        <label>
+          Full date of birth:
+          <input type="date" name="Full date of birth"> 
+        </label>
+        <br><br>
+        <label>
+          I accept the term of the agreement
+          <input type="checkbox" name="I accept the term of the agreement">
+        </label>
+      </div>
+      <div class="registration">
+        <div class="text_border">Registration</div>
+        <label>
+          E-mail:
+          <input type="E-mail" name="E-mail" value="email@example.com" pattern="email@example.com" size="30" required>
```

Comment:
```suggestion
          <input 
            type="E-mail" 
            name="E-mail" 
            value="email@example.com" 
            pattern="email@example.com" 
            size="30" required
          >
```
якщо в тебе 3 або більше атрибута пиши коже з нового рядка

Reply:
тут і в інших місцях теж

Reply:
Ок, зрозумів, дякую))